samples/SkiaMultiplatformSample/build.gradle.kts

```diff
@@ -61,30 +64,20 @@ val unzipTask = tasks.register("unzipWasm", Copy::class) {
 }
 
 kotlin {
-    val targets = mutableListOf<org.jetbrains.kotlin.gradle.plugin.mpp.KotlinNativeTarget>()
-
-    if (hostOs == "macos") {
-        val nativeHostTarget = when (host) {
-            "macos-x64" -> macosX64()
-            "macos-arm64" -> macosArm64()
-            else -> throw GradleException("Host OS is not supported yet")
+    if (hostOs == "macos") { //todo maybe redundant check
```

Comment:
I think it's indeed redundant. And having it disabled on other OS might even be not desirable, because changing some common code API can affect disabled macOs source sets and IDE won't complain about it.

For example in compose project we have all targets declared unconditionally and the build seems to be working on mac os and windows.